src/OrleansRuntime/Messaging/GatewayAcceptor.cs

```diff
@@ -10,18 +11,21 @@ internal class GatewayAcceptor : IncomingMessageAcceptor
         private readonly Gateway gateway;
         private readonly CounterStatistic loadSheddingCounter;
         private readonly CounterStatistic gatewayTrafficCounter;
+        private readonly GlobalConfiguration globalConfig;
 
         internal GatewayAcceptor(
             MessageCenter msgCtr,
             Gateway gateway,
             IPEndPoint gatewayAddress,
             MessageFactory messageFactory,
-            SerializationManager serializationManager)
+            SerializationManager serializationManager,
+            GlobalConfiguration globalConfig)
```

Comment:
in theory ClusterId and HasMultiClusterNetwork properties could be moved to ILocalSiloDetails, so no need to reference the uber global config (which we want to get rid of eventually with the SiloBuilder)